docs/pages/modals/index.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        Simple modals break the user’s attention away from the main UI in order to focus attention on a singular action or task. They can contain alerts about important information and allow the user to edit data.
+      </p>
+    </DocsPageSection>
+
+    <DocsPageSection title="Layout" anchor="#layout">
+      <p>
+        Modals have 3 sections that are always present: <strong>title, content</strong>, and <strong>actions</strong>.
+      </p>
+      <p>
+        Optional:
+      </p>
+      <ul>
+        <li>Text prop below title</li>
+        <li>Secondary button</li>
+      </ul>
+      <img src="./modal-types.png">
```

Comment:
Recommend splitting this into two images so that it will wrap on smaller screens:

![image](https://user-images.githubusercontent.com/2367265/87357814-8685ee80-c519-11ea-9197-f12470122a07.png)

You can also use the `<DocsShow>` component to add a border:

```suggestion
      <DocsShow>
        <img src="./modal-type-1.png">
        <img src="./modal-type-2.png">
      </DocsShow>
```

Reply:
Cool, done and updated the PR